format/index.md

```diff
@@ -472,14 +488,14 @@ multiple times.
 
 As discussed above, the document **MAY** be extended to include
 meta-information as `"meta"` members in several locations: at the top-level,
-within resource objects, within link objects, and within linkage objects.
+within resource objects, within relationship objects, and within linkage objects.
```

Comment:
Cool. I'm not thrilled with the name either, so let me know if you think of something better!